PySDM/physics/saturation_vapour_pressure/bolton_1980.py

```diff
@@ -0,0 +1,42 @@
+"""
+[Bolton 1980](https://doi.org/10.1175/1520-0493(1980)108<1046:TCOEPT>2.0.CO;2)
+"""
+
+import numpy as np
+
+
+class Wexler1976:
+    def __init__(self, _):
+        pass
+
+    @staticmethod
+    def pvs_Kelvins(const, T):
+        """valid for 0 <= T <= 100 C, eq (9)"""
+        return (
+            np.exp((T**-2) * const.W76W_G0)
+            + ((T**-1) * const.W76W_G1)
+            + (T * const.W76W_G2)
+            + ((T**1) * const.W76W_G3)
+            + ((T**2) * const.W76W_G4)
+            + ((T**3) * const.W76W_G5)
+            + ((T**4) * const.W76W_G6)
+            + (np.log(T) * const.W76W_G7)
+        ) * const.W76W_G8
+
+    @staticmethod
+    def ice_Kelvins(const, T):  # pylint: disable=unused-argument
+        return np.nan
+
+
+class Bolton1980:
+    def __init__(self, _):
+        pass
+
+    @staticmethod
+    def pvs_Celsius(const, T):
+        """valid for -30 <= T <= 35 C, eq (10)"""
+        return (const.B80W_G0 * np.exp) * ((const.B80W_G1 * T) / (T + const.B80W_G2))
```

Comment:
`exp` needs an argument here, running this file under pytest will help to debug before committing: https://github.com/open-atmos/PySDM/blob/main/tests/unit_tests/physics/test_saturation_vapour_pressure.py (with `plot=True`, there will be a plot shown helping to see if the coefficients have correct units, etc by comparing with other already implemented formulae for svp)